README.rst

```diff
@@ -1214,6 +1214,25 @@ Choose from the list of available rules:
 
   *Risky rule: this fixer may change functions named ``setUp()`` or ``tearDown()`` outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
 
+* **php_unit_short_will_return**
+
+  Usage of ``->will($this->returnValue(..))`` statements must be replaced by
+  it's shorter equivalent ``->willReturn(...)``.
+
+  *Risky rule: risky when PHPUnit classes are overridden or not accessible, or when project has PHPUnit incompatibilities.*
+
+  Configuration options:
+
+  - ``returnArgument`` (``false``, ``true``): fix ``returnArgument`` occurrence; defaults
+    to ``true``
+  - ``returnCallback`` (``false``, ``true``): fix ``returnCallback`` occurrence; defaults
+    to ``true``
+  - ``returnSelf`` (``false``, ``true``): fix ``returnSelf`` occurrence; defaults to ``true``
+  - ``returnValue`` (``false``, ``true``): fix ``returnValue`` occurrence; defaults to
+    ``true``
+  - ``returnValueMap`` (``false``, ``true``): fix ``returnValueMap`` occurrence; defaults
```

Comment:
to match with the other options in the project, can you use `snake_case` instead of `camelCase`?